SPECS/openssl/openssl.spec

```diff
@@ -365,6 +367,9 @@ rm -f %{buildroot}%{_sysconfdir}/pki/tls/ct_log_list.cnf.dist
 %postun libs -p /sbin/ldconfig
 
 %changelog
+* Thu May 23 2024 Juan Camposeco <juan.camposeco@gmail.com> - 1.1.1k-32
```

Comment:
Please put the fast-track change on the same release as in `fasttrack/2.0`. The ordering of the patches above is not that important, but we want to maintain a consistent changelog in our published packages.

Reply:
Then, should I bump Tobias's release 31->32 and keep mine in 31, please confirm

Reply:
Yup, exactly that.